workers/unity/Packages/io.improbable.gdk.debug/NetStatViewer.cs

```diff
@@ -0,0 +1,164 @@
+using System.Collections.Generic;
+using System.Linq;
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Core.NetworkStats;
+using Unity.Entities;
+using UnityEditor;
+using UnityEditor.UIElements;
+using UnityEngine.UIElements;
+
+namespace Improbable.Gdk.Debug
+{
+    internal class NetStatViewer : EditorWindow
+    {
+        private const string WindowUxmlPath = "Packages/io.improbable.gdk.debug/Templates/NetStatsWindow.uxml";
+        private const string UpdateRowUxmlPath = "Packages/io.improbable.gdk.debug/Templates/UpdateRow.uxml";
+
+        private World selectedWorld;
+        private NetworkStatisticsSystem netStatSystem;
+        private List<(string Name, uint ComponentId)> spatialComponents;
+        private Dictionary<int, VisualElement> listElements;
+
+        [MenuItem("SpatialOS/Window/Network Analyzer", false)]
+        public static void ShowWindow()
+        {
+            var inspectorWindowType = typeof(EditorWindow).Assembly.GetType("UnityEditor.InspectorWindow");
+            var window = GetWindow<NetStatViewer>(inspectorWindowType);
+            window.titleContent.text = "Network Analyzer";
+            window.Show();
+        }
+
+        private void OnEnable()
+        {
+            // Generate list of types
+            spatialComponents = ComponentDatabase.ComponentsToIds
+                .Select(pair => (pair.Key.DeclaringType.Name, pair.Value))
+                .ToList();
+
+            // Load UI
+            SetupUI();
+            SetupWorldSelection();
+        }
+
+        // Update UI at 10 FPS
+        private void OnInspectorUpdate()
+        {
+            SetupWorldSelection();
+            RefreshUpdates();
+        }
+
+        private void SetupUI()
+        {
+            // Load main window
+            var windowTemplate = AssetDatabase.LoadAssetAtPath<VisualTreeAsset>(WindowUxmlPath);
+            windowTemplate.CloneTree(rootVisualElement);
+
+            // Load update row
+            var itemTemplate = AssetDatabase.LoadAssetAtPath<VisualTreeAsset>(UpdateRowUxmlPath);
+
+            // Clone row for each component, fill list
+            var scrollView = rootVisualElement.Q<ScrollView>("updatesContainer");
```

Comment:
For future proofing (i.e. - when we do non-update things) lets rename the container `updatesContainer` -> `dataContainer` ?

Reply:
Discussed offline, this wont be reused for non-update things